typography.scss

```diff
@@ -21,7 +20,7 @@ $vui-base-fontSize: 19px;
 	font-family: 'Lato', 'Lucida Sans Unicode', 'Lucida Grande', sans-serif;
 	font-size: 1rem;
 	font-weight: 300;
-	line-height: 1.5rem;
+	line-height: 1.5;
```

Comment:
Cool.  I looked into this a bit more, and found a few more places recommending the unitless approach for line-height.